src/SignalR/common/Protocols.MessagePack/src/Protocol/MessagePackHubProtocolWorker.cs

```diff
@@ -79,8 +79,12 @@ private HubMessage CreateInvocationMessage(ref MessagePackReader reader, IInvoca
         }
 
         var target = ReadString(ref reader, binder, "target");
+        if (string.IsNullOrEmpty(target))
```

Comment:
Not really, it would have thrown later at the `binder.GetParameterTypes(target)` call, which would result in slightly different exception behavior. This just provides a more accurate exception message.